test/e2e/bootstrapper_test.go

```diff
@@ -110,3 +130,18 @@ func getSvcState(svcName string) (svc.State, error) {
 	}
 	return status.State, nil
 }
+
+// removeFileIfExists removes the file given by 'path', and will not throw an error if it does not exist
+func removeFileIfExists(t *testing.T, path string) {
+	err := os.Remove(path)
+	if err != nil {
+		assert.Truef(t, os.IsNotExist(err), "could not remove file %s: %s", path, err)
+	}
+}
+
+// isKubeletRunning checks if the kubelet was able to start sucessfully
+func isKubeletRunning(t *testing.T, logPath string) bool {
+	buf, err := ioutil.ReadFile(logPath)
+	assert.Nil(t, err)
+	return strings.Contains(string(buf), "Started kubelet")
```

Comment:
Not sure, if guarantees kubelet has started? Do you want to wait for some time before kubelet crashes?

Reply:
How would we capture it crashing? 

Also, I feel like this test should only capture whether or not the kubelet was able to start. If we rely on this test to see if it stays up, the test will be flaky as we will be waiting an arbitrary amount of time for it to fail, and not checking past that time. I feel like there are better ways to test the stability of a service.

Reply:
> I feel like there are better ways to test the stability of a service.

Like what? If you think there is a better way to do it, by all means let's use that.

Reply:
It would be through a long (multiple days -> weeks) run, with health monitoring. The point I  was trying to make, is that this isn't something an end to end test is suited for.

Reply:
I agree with @sebsoto on this